docs/spec_draft.md

```diff
@@ -189,6 +191,45 @@ logical operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.ceil
+
+`stablehlo.ceil(operand) -> result`
+
+### Semantics
+
+Performs element-wise ceil of `operand` tensor and produces a `result` tensor,
+         where ceil of a floating-point value is defined as the smallest integer
+         greater than or equal to the value. Implements the rounding to integral
+         towards positive infinity operation from the IEEE-754 specification.
```

Comment:
"where ceil of a floating-point value is defined as the smallest integer greater than or equal to the value". Let's drop this, now that we have a reference to IEEE-754.